### PR TITLE
Association trigger

### PR DESCRIPTION
**Pull request by orion-belt**
Support for Association trigger

### CODE REVIEW

File: src/common/utils/fqdn.cpp [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      boost::asio::ip::tcp::endpoint endpoint = *it;
+      address                                 = endpoint.address().to_string();
+      port                                    = endpoint.port();
+      Logger::spgwu_app().debug(
+          "Resolve a DNS (name %s, protocol %s): Ip Addr %s, port %u",
+          host_name.c_str(), protocol.c_str(), address.c_str(), port);
+      if (endpoint.address().is_v4())
+        addr_type = 0;
+      else
+        addr_type = 1;
```
Comment (lionelgo) [inline src/common/utils/fqdn.cpp]: You may use constants, or use is_ipv4 or is ipv6 boolean

File: src/oai_spgwc/pgw_app.cpp [hunk 2]
```diff
@@ -306,9 +307,91 @@ pgw_app::pgw_app(const std::string& config_file)
...
+            sxa_asc.get()->get_msg_name());
+      } else {
+        Logger::pgwc_app().debug("Association request sent");
+      }
+    } else {
+      Logger::pgwc_app().warn("TODO start_association() node_id IPV6");
+    }
+  }
+}
+//------------------------------------------------------------------------------
```
Comment (lionelgo) [inline src/oai_spgwc/pgw_app.cpp:406]: If UP node not ready, some logic is necessary to do some retries ?

Comment (rohanrkharade) [reply src/oai_spgwc/pgw_app.cpp:406]: @lionelgo 
This is addressed in [latest commit.](https://github.com/OPENAIRINTERFACE/openair-spgwc/pull/29/commits/529206b5acc955ab2e9ca6bca170f66d17a18f92)

File: src/common/utils/fqdn.cpp (hunk 1 above)
Comment (rohanrkharade) [reply src/common/utils/fqdn.cpp]: @lionelgo 
This is addressed in [latest commit.](https://github.com/OPENAIRINTERFACE/openair-spgwc/pull/29/commits/105cd4d83b5954dbe2dd4e05fddf452dc60c0114)

**Review by lionelgo — APPROVED** [review]

**Comment by rdefosse** [comment]
@orion-belt 

`https://jenkins-oai.eurecom.fr/job/OAI-CN-SPGWC/72/artifact/test_results_oai_spgwc.html` --> 2 files do not follow formatting policy

**Review by rdefosse — APPROVED** [review]
All files are properly formatted.

**Pull request merged**